torchmd/forces.py

```diff
@@ -90,16 +90,10 @@ def compute(
         toNumpy=True,
         calculateForces=True,
     ):
-        if not explicit_forces and not pos.requires_grad:
```

Comment:
I'm fine with the rest of the changes but do you really need to remove those checks for vmap?